Mathlib/Geometry/Manifold/VectorBundle/Basic.lean

```diff
@@ -621,6 +628,72 @@ instance Bundle.Prod.smoothVectorBundle : SmoothVectorBundle (F₁ × F₂) (E
       simp only [Trivialization.baseSet_prod, mfld_simps]
       mfld_set_tac
 
+/-- For smooth vector bundles `E₁` and `E₂` over a manifold `B`, the natural projection from the
+total space of `E₁ ×ᵇ E₂` to the total space of `E₁` is smooth. -/
+theorem Bundle.Prod.smooth_fst :
```

Comment:
I'm a little bit surprised you need the fact that the bundles are smooth here: after all, when you read the map in charts, it should be the map `(b, f_1, f_2) -> (b, f_1)`, which is always smooth. Or am I missing something?

(Especially because you don't need it below)

Reply:
(Working on it!)